docs/pvcscale.md

```diff
@@ -0,0 +1,197 @@
+# PVCScale Workload
+
+The PVCScale workload playbook is `workloads/pvcscale.yml` and will run the pvcscale workload on your cluster.
+
+PVCScale workload stresses storage backend and determines how fast Pods are started depending on storage backend. 
+Each Pod requires a PVC to be allocated and attached.
+
+PVCScale test creates a desired number of Pods and attaches one PVC to each Pod, it supports 
+only dynamic PVC allocation using storage class and in order for this test to 
+work it is a requirement to have a working storage class which can be used to allocate a PVC.
+
+At end of test, the time necessary for Pods to reach `READY` state with PVC
+in `Bound` status is measured, and based on this time we can compare start time
+for same number of Pods/PVCS for cases when only difference is storage class
+used to allocate PVCS. 
+
+The situation when Pod is in `READY` state and  with PVC in `Bound` status is considered as state where PVC is
+ready for consumption by application Pod.
+
+Different storage backends behaves differently when it comes to PVC scalability and it is necessary to have 
+repeatable results which can answer the question what storage backend provides optimal PVCs scale and density results for
+scenario where many Pods/PVCs are necessary to start in short period of time.
+
+Running from CLI:
+
+```sh
+$ cp workloads/inventory.example inventory
+$ # Add orchestration host to inventory
+$ # Edit vars in workloads/vars/pvcscale.yml or define Environment vars (See below)
+$ time ansible-playbook -vv -i inventory workloads/pvcscale.yml
+```
+
+## Environment variables
+
+### PUBLIC_KEY
+Default: `~/.ssh/id_rsa.pub`
```

Comment:
Each of these (Each env var default) needs 2 spaces after the "default" this will cause a break to be between the Default and the text below the default.  Currently markdown consumes the newline without these two spaces, take a look at the other readme files to see an example.

Reply:
@akrzos fixed,